### PR TITLE
Cis benchmark 4.3.3

### PR DESCRIPTION
**Pull request by ndowmon**
This will allow us to answer 4.3.3 -> 4.3.7

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +10,12 @@ and this project adheres to
 
 ### Added
 
+- Added the following properties to `azure_postgresql_server`:
+  - `configurations.log_checkpoints`
+  - `configurations.log_connections`
+  - `configurations.log_disconnections`
+  - `configurations.connection_throttling`
+  - `configurations.log_retention_days`
```
Comment (aiwilliams) [inline CHANGELOG.md:18]: I understand from the code that there are multiple `PostgreSQLServerConfiguration` objects, but I wonder if the user would be best served by a singular name, such as `configuration.log_checkpoints`. When I saw the screen full of configuration values, I did not think about how they were represented in data, only that the server had a configuration.

File: src/steps/resource-manager/databases/postgresql/index.test.ts [hunk 2]
```diff
@@ -1,489 +1,181 @@
...
-      directoryId: '4a17becb-fb42-4633-b5c8-5ab66f28d195',
-      subscriptionId: '87f62f44-9dad-4284-a08f-f2fb3d8b528a',
-      developerId: 'keionned',
+afterEach(async () => {
+  if (recording) {
+    await recording.stop();
+  }
+});
+
+describe('rm-database-postgresql-servers', () => {
```
Comment (aiwilliams) [inline src/steps/resource-manager/databases/postgresql/index.test.ts:28]: Liking the refactoring on these tests 👍

File: src/getStepStartStates.test.ts [hunk 3]
```diff
@@ -137,7 +137,9 @@ describe('getStepStartStates', () => {
       [STEP_RM_DATABASE_MYSQL_DATABASES]: { disabled: true },
       [sqlDatabaseSteps.DATABASES]: { disabled: true },
       [sqlDatabaseSteps.SERVER_FIREWALL_RULES]: { disabled: true },
-      [STEP_RM_DATABASE_POSTGRESQL_DATABASES]: { disabled: true },
+      [postgreSqlDatabaseSteps.SERVERS]: { disabled: true },
+      [postgreSqlDatabaseSteps.DATABASES]: { disabled: true },
+      // [STEP_RM_DATABASE_POSTGRESQL_DATABASES]: { disabled: true },
```
Comment (aiwilliams) [inline src/getStepStartStates.test.ts:142]: These commented codes can be deleted, eh?

Comment (ndowmon) [reply src/getStepStartStates.test.ts:142]: Yes... I didn't notice them before opening this PR but they're deleted in the next (#254)

File: CHANGELOG.md (hunk 1 above)
Comment (ndowmon) [reply CHANGELOG.md:18]: Good suggestion. 

- [ ] Rename `azure_postgresql_server.configurations.*` -> `azure_postgresql_server.configuration.*`

**Pull request merged**